spec/lib/miq_automation_engine/miq_ae_service_spec.rb

```diff
@@ -116,6 +116,8 @@ module MiqAeServiceSpec
           Datacenter
           ManageIQ::Providers::BaseManager
           ManageIQ::Providers::PhysicalInfraManager
+          ManageIQ::Providers::Lenovo::PhysicalInfraManager
+          ManageIQ::Providers::Lenovo::Provider
```

Comment:
@gmcculloug Can you review this?  I feel like these should not be excluded, but not sure what you think.